### DISCUSSION TITLE
Disable sentencizer

### DISCUSSION THREAD

**Discussion by firejake308** [medspacy/medspacy#209] [Q&A]
Is it possible to disable the sentencizer but keep the context matcher of the default pipeline? The pipeline is running pretty slow for me, and I'm trying to think of ways to speed it up, since all I really want is some NER and enough context to know which of the entitities are negated. I tried calling `nlp.disable_pipe('medspacy_pyrush')` but then the context module started throwing errors about not having a sentencizer

**Comment by burgersmoke** [comment]
Hello @firejake308 , yes you can disable the medspacy sentence boundary detection (i.e., "medspacy_pyrush"), but you are correct that if you do not specify another method for detecting sentences, context will not work since it needs sentences to determine scope of applying modifiers.  

Here's a quick code example of just one of the many ways that you can disable the default medspacy sentence solution and use the default spacy solution instead.  Does this give you what you are asking for?

```
nlp = medspacy.load()
print(f"""Pipe names before changing: {nlp.pipe_names}""")

# now let's remove pyrush
nlp.remove_pipe("medspacy_pyrush")

print(f"""Pipe names after removing PyRush: {nlp.pipe_names}""")

# now let's add the default spacy sentence boundary detection, and make sure it is first
nlp.add_pipe("sentencizer", first = True)

print(f"""Pipe names after adding default spacy sentencizer: {nlp.pipe_names}""") 
```
